### ISSUE TITLE
Implement RSA algorithm

### ISSUE THREAD

**Issue by robbert229** [robbert229/jwt#6]
Currently there is no support for the RSA family of algorithms.

**Comment by alexandrevicenzi** [comment]
@robbert229 maybe use https://golang.org/pkg/crypto/rsa/ right?

Looking into the code it's just a small change at `algorithms.go`. If so I could send a PR.

**Comment by robbert229** [comment]
That is correct. It should be a relatively simple feature. The PR would be appreciated!

**Comment by alexandrevicenzi** [comment]
Ok, I'll take a look :)
